balboa-core/src/main/scala/com/socrata/balboa/metrics/data/impl/Cassandra11QueryImpl.scala

```diff
@@ -44,12 +42,15 @@ class Cassandra11QueryImpl(context: AstyanaxContext[Keyspace]) extends Cassandra
   def getAllEntityIds(recordType: RecordType, period: Period): Iterator[String] = {
     fastfail.proceedOrThrow()
     try {
-      val retVal: Iterator[String] = context.getEntity.prepareQuery(Cassandra11Util.getColumnFamily(period, recordType))
-        .setConsistencyLevel(ConsistencyLevel.CL_ONE)
-        .withRetryPolicy(new ExponentialBackoff(250, 5)) // initial, max tries
```

Comment:
I'm going to set the LoggingPolicy on as the RetryPolicy in Cassandra11Util.scala, rather than here (as it would only affect that specific query).